web/webpack.config.js

```diff
@@ -68,10 +74,6 @@ const copyWebpackCommands = [
     from: `${WEB_STATIC_ROOT}/pwa/`,
     to: `${DIST_ROOT}/public/pwa/`,
   },
-  {
-    from: `${STATIC_ROOT}/../custom/homepages/v2/announcement`,
-    to: `${DIST_ROOT}/announcement`,
-  },
```

Comment:
Still need this.

I can give you a mock copy of the custom homepage files so you can build `dev-server` the same way.  We didn't handle `CUSTOM_HOMEPAGE=false` for contributors well, so that could be the cause for confusion.

Reply:
Good to know. No worries.  I can figure something to either ignore this or configure it to not be a thing when that option is false, now that I know that this is how it should behave.